traefik/templates/service.yaml

```diff
@@ -3,7 +3,7 @@
 {{ $tcpPorts := dict }}
 {{ $udpPorts := dict }}
 {{- range $name, $config := .Values.ports }}
-  {{- if eq ($config.protocol | toString) "UDP" }}
+  {{- if eq (toString $config.protocol) "UDP" }}
```

Comment:
also here - what's that for?

Reply:
https://github.com/containous/traefik-helm-chart/pull/245#discussion_r474518059